### ISSUE TITLE
Need to have UI for pupils to login and conduct their own assessment

### ISSUE THREAD

**Issue by icdev2dev** [icdev2dev/luften#2]
The UI currently exposes an admin interface. However no end user (pupil) functionality. This can be simple to begin with.

- when creating an user, add an additional field called password and enter some simple text into that field...nothing fancy to begin with.
- when the end user logins, let the end user enter both username and password. If both match, then let the end user in.

In this, there is no Home/Users/Administration etc tabs  because this is for the end pupil.

**Comment by A-1104** [comment]
The issues has been solved and has already been updated into the code

**Comment by tmknpp** [comment]
we just did a stresstest with about 10 people and it didn't work. after the first few people logged in, login was extremely slow for the rest. messages were only processed for the first 3 or so as well. as it stands right now, we will have to cancel the experiment tomorrow. i copied the logs from terminal :

https://docs.google.com/document/d/1USUqPMV8ovICjxXa0vEQgHZutqvGwcSy_Hvz2eOmGBg/edit

do you think paying for better hardware on the server could fix the problem?

**Comment by A-1104** [comment]
It pretty late here let me go to the office tmm and check the issue withi
sir

**Comment by tmknpp** [comment]
yes, of course

**Comment by A-1104** [comment]
And what abt we have a meet around 10 CEST time?

**Comment by tmknpp** [comment]
yes we can. we will cancel the experiment for tomorrow though, because it's in slovakia and i would need to leave the house around 6:15 to get there on time.

**Comment by A-1104** [comment]
Sure lets wait for 1,2 days more and fix the issues

**Comment by tmknpp** [comment]
Ah, I saw I commented on the other issue.. Kamila still wants us to come to Slovakia tomorrow, we will try to do the testing with 2-3 pupils at the same time only. So I'm not going to be available at 10:00 CEST for a meeting, but can still read and write here until about 10:30. Also, we still have another testing session with Tereza's class, so maybe we'll be able to solve the issues until then

**Comment by A-1104** [comment]
Ok, sounds good.

**Comment by A-1104** [comment]
can you meet me when you do the research and i need to know what it is going

**Comment by tmknpp** [comment]
maybe I can install it on a few computers locally after all and then we shouldn't have problems, I need to see when I get to the school. I think I will be busy during the experiment, but if there is someone else there then it would maybe be possible. I will tell you

**Comment by A-1104** [comment]
ok no worries you can do that but just contact us as soon as you can

**Comment by A-1104** [comment]
as we just checked for our side and there is no issue so I think when you go to the school and before starting the experiment nu can just have a meet with us for like 10 mins and lets see whats going on

**Comment by tmknpp** [comment]
i will meet kamila soon now and need to get on a new bus after, i could do a quick meeting from the new bus with internet hotspot from my phone

**Comment by A-1104** [comment]
Ok, sounds good. or you can do a meet once you meet kamila and have time
both are ok to us

**Comment by tmknpp** [comment]
I'd have time now!

**Comment by A-1104** [comment]
Ok then lets have a meeting in 5 min

**Comment by tmknpp** [comment]
ok

**Comment by A-1104** [comment]
I am in the meet 
where are you
